pyclaragenomics/samples/sample_cudapoa

```diff
@@ -38,11 +38,12 @@ def run_cudapoa(groups, msa, print_output):
 
     # Create batch
     max_sequences_per_poa = 100
-    gpu_mem_per_batch = 0.9 * free
```

Comment:
let's keep the variable name around so it's easier for the user to understand what's being passed into the API